### ISSUE TITLE
Webserver home page not showing servers

### ISSUE THREAD

**Issue by rianadon** [typicode/hotel#95]
I just installed hotel today, and it looks very cool!
However, after I started hotel and added a server, the webpage stills displays "Welcome, please enjoy your stay! Use hotel command-line to add servers".

Interestingly though, once I run `hotel stop`, the list with the server's name appears.
I can also still use hotel by navigating to `localhost:2000/project`.

This happens in both the latest versions of Chrome and Firefox on Windows 10.

Thank you!

**Comment by typicode** [comment]
Hi @rianadon,

Thanks for the comment and reporting this issue.

I can't seem to reproduce (tried with Chrome, Firefox on Windows 10). If you open the browser dev tools, do you see some errors?

**Comment by rianadon** [comment]
Thanks for looking into this!

There aren't any errors, but vuex is complaining that an action is not a function (see below).

![screenshot 5](https://cloud.githubusercontent.com/assets/18223213/14943765/5e5f1018-0f96-11e6-8628-23ca674db353.png)

Could this be the error?

Upon further inspection, the problem seems to be with the `/_/events` url, which completes when I manually stop hotel. I tried listening in on the event stream through following code in devtools, then tried adding new hotel servers.

```javascript
var source = new EventSource('/_/events');
source.addEventListener('message', function(e) {
  console.log(e.data);
}, false);
```

When adding new servers nothing appeared, but once I ran `hotel stop`, the console filled up with messages.

Then, I used telnet to fake an http request to `/_/events`, and I got text!
So maybe the Node.js script is formatting the text stream incorrectly?

**Comment by typicode** [comment]
Thanks for doing these tests. Maybe it's related to the version of Node used? What's your version?
I assume that you get this error by going to `http://localhost:2000` and not `http://hotel.dev`.

**Comment by rianadon** [comment]
Currently I'm on 5.3.0. And yes, I'm using `http://localhost:2000`.

**Comment by rianadon** [comment]
I just installed 6.0.0 and I'm still encountering the same problem.
Thanks for the suggestion though!

**Comment by typicode** [comment]
Weird. I did the test with `6.0.0` too, same OS and browsers.

**Comment by rianadon** [comment]
I forgot to mention this as I deemed it unimportant, but the computer on which hotel is not working is in fact running a Windows Insiders Build (160422).

Since I tried installing hotel on a regular Windows build (same node version) and it worked, I guess this is either somehow related to my specific machine or the different build of Windows (but I have no idea why :confused: ).

Moving on, the `/_/events` endpoint is definitely the problem. It displays as loaded in the chrome dev tools on the working computer but displays as pending on the computer with the Insiders build.

I'll update this comment as I do more testing, and I guess if you are able to you could try running hotel on a computer with a Windows Insider Preview.

**Comment by rianadon** [comment]
As I mentioned in my previous comment, this seems to be related to an issue with Node.js and the Windows Insider build and not hotel. Thank you so much for trying to help resolve the issue!

**Comment by rianadon** [comment]
If anyone's curious the new issue is at https://github.com/nodejs/node/issues/6541.

**Comment by typicode** [comment]
Awesome job figuring everything out (didn't even know about Windows Insider), thank you!

**Comment by rianadon** [comment]
You're welcome.
